src/components/WizardFooter/WizardFooter.tsx

```diff
@@ -0,0 +1,129 @@
+import { Button } from '@amsterdam/asc-ui';
+import { ChevronLeft } from '@amsterdam/asc-assets';
+import React, { ComponentProps } from 'react';
+import {
+	ButtonPanelStyle,
+	CancelButtonStyle,
+	LeftActionStyle,
+	PreviousButtonStyle,
+	RightActionStyle,
+	WizardButton,
+} from './WizardFooterStyles';
+
+type ButtonConfig = {
+	visible?: boolean;
+	disabled?: boolean;
+	label?: string;
+	onClick: ComponentProps<typeof Button>['onClick'];
+	dataTestId?: string | null;
```

Comment:
Why null? it can already be undefined. Better to type as optional string IMO.
```
dataTestId?: string;
```

Reply:
Done